### ISSUE TITLE
brew install not working

### ISSUE THREAD

**Issue by ullstrm** [SwiftGen/SwiftGen#513]
I'm having problems installing SwiftGen via homebrew. This is the log I get. SwiftGen is the only formulae that has caused problems so far. Any ideas?

This is the log:

```
$ brew install swiftgen
==> Cloning https://github.com/SwiftGen/SwiftGen.git
Updating /Users/xxx/Library/Caches/Homebrew/swiftgen--git
==> Checking out tag 5.3.0
HEAD is now at 73ac0e4 Merge pull request #395 from SwiftGen/fix/homebrew
HEAD is now at 73ac0e4 Merge pull request #395 from SwiftGen/fix/homebrew
==> gem install bundler
==> bundle install --without development release
==> bundle exec rake cli:install[/usr/local/Cellar/swiftgen/5.3.0/bin,/usr/local
Last 15 lines from /Users/xxx/Library/Logs/Homebrew/swiftgen/03.bundle:
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/cli/exec.rb:28:in `run'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/cli.rb:424:in `exec'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/cli.rb:27:in `dispatch'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/cli.rb:18:in `start'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/exe/bundle:30:in `block in <top (required)>'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/gems/bundler-1.16.5/exe/bundle:22:in `<top (required)>'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/bin/bundle:22:in `load'
/private/tmp/swiftgen-20180928-11853-1uwgnku/gem_home/bin/bundle:22:in `<main>'
Tasks: TOP => cli:install => cli:build
(See full trace by running task with --trace)

READ THIS: https://docs.brew.sh/Troubleshooting
```

If I run it with an empty homebrew-cache I also get this:

```
...
==> Cloning https://github.com/SwiftGen/SwiftGen.git
Cloning into '/Users/xxx/Library/Caches/Homebrew/swiftgen--git'...
Note: checking out '73ac0e487dda941eb4e52ed3b96b652515f1bee5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

==> Checking out tag 5.3.0
....
```

**Comment by AliSoftware** [comment]
Hi @ullstrm 

Thanks a lot for reporting this issue!

* SwiftGen 5.3.0 was released a while ago, and the brew formula thus required/used Xcode 9.2 to build back then. Maybe it can't find Xcode 9.2 in your machine and that's why it fails? (We print an explicit error message in those cases, but maybe it's hidden in the noice since homebrew seems to only log the last 15 lines…) maybe using `--trace` you'd get more info about what actually fails?
* We actually know that we'll have to update our brew formula because since SwiftGen 5.3.0, the files have moved in the repo, so you won't be able to `brew install swiftgen --head` either to try to install from the latest commit. The latest formula posted on homebrew back in the day we released 5.3.0 is not compatible with the head / master and we'll have to update it soon

That being said, we actually are right now in preparation to release SwiftGen 6.0 (release should be done this week-end) — see https://github.com/SwiftGen/SwiftGen/issues/499, in which we actually also mention the updates we'll have to do to update our homebrew formula in the process — so maybe your best move is to wait for that 6.0 release and re-test with it after this week-end, once we've published the new homebrew formula for 6.0 and hopefully fixed it all!
